agent/bench-scripts/pbench-trafficgen

```diff
@@ -97,6 +97,24 @@ function kill_trex() {
 	fi
 }
 
+function convert_number_range() {
+        # converts a range of cpus, like "1-3,5" to a list, like "1,2,3,5"
+        local cpu_range=$1
+        local cpus_list=""
+        local cpus=""
+        for cpus in `echo "$cpu_range" | sed -e 's/,/ /g'`; do
+                if echo "$cpus" | grep -q -- "-"; then
+                        cpus=`echo $cpus | sed -e 's/-/ /'`
+                        cpus=`seq $cpus | sed -e 's/ /,/g'`
```

Comment:
Karl, this line should be:
 cpus=`seq -s ' ' $cpus | sed -e 's/ /,/g'`

Reply:
[jianzzha@jianzzha nfv_perf]$ cpus="1 3"
[jianzzha@jianzzha nfv_perf]$ seq $cpus | sed -e 's/ /,/g'
1
2
3
[jianzzha@jianzzha nfv_perf]$ seq -s ' ' $cpus | sed -e 's/ /,/g'
1,2,3

The comma list is what you want to get, right?